tx-source/dash_core.php

```diff
@@ -647,6 +647,7 @@
 $Definition['Save Captured Definitions'] = 'Save Captured Definitions';
 $Definition['Save Colors'] = 'Save Colors';
 $Definition['Search by user or role.'] = 'Search for users by name or email, optionally using % as a wildcard. You can also search by user ID, the name of a role, or &ldquo;banned&rdquo;.';
+$Definition['Search results of members.'] = 'Search results of members.';
```

Comment:
It does not sound correct to me.
@charrondev could you spell it better?

Reply:
Stéphane picked the text, and I think it looks fine. Like you have suggested to me privately, I'll ask Adam to approve it.